platform.ini

```diff
@@ -0,0 +1,8 @@
+[COMMON]
+main_url = https://github.com/YetAnotherPackageInstaller
+
+[DARWIN]
```

Comment:
Is there a reason for a MacOS and Debian package location in YAPI Origin? I thought this was a single platform for debian. Please explain this config.

Reply:
Maybe in a future developing this can be useful, I didn't think about yapi as multiplatform but who knows